### DISCUSSION TITLE
CW not working

### DISCUSSION THREAD

**Discussion by NickmDrummer** [aws/codewhisperer-command-line-discussions#80] [Support ticket]
### Checks

- [X] I ran `cw doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `cw restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

Working.

### Actual behaviour

Not working.

### Steps to reproduce

1- Install Code Whisperer
2- Open the app and verify that all is ok.
3- Try in iTerm2 (Not Working)
4- Run ``cw doctor`` (Results are OK)
5- Uninstall and reinstall many times.
6- Not working

**Comment by brendanfalk** [comment]
Hey Nick - could you please run `cw issue` to report the issue? That will give us more context so we can help

**Comment by grant0417** [comment]
Going to close this since since it is stale, feel free to reopen the issue if you still have issues.
